### DISCUSSION TITLE
Can ES indexes be created by the hour?

### DISCUSSION THREAD

**Discussion by kanggeyyds** [apache/skywalking#12650] [Q&A]
Can ES indexes be created by the hour? 
Because the daily segment index data volume is too large

**Comment by wu-sheng** [comment]
That should use Elasticsearch sharding, rather than creating by hour. Please use Elasticsearch properly.

**Reply by kanggeyyds** [reply]
My daily index is too large, and the number of ES clusters is too small. Calculating based on ES shards of 50G per node already allocates 5-6 shards. How can I optimize without considering expanding the ES nodes

**Reply by wu-sheng** [reply]
Sharding is as same as the new index. If you can't survive from the more shards, you would not work for more indices.

**Reply by kanggeyyds** [reply]
Okay, thank you for your answer
